x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/action_start/start_button_modal.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiConfirmModal, EuiOverlayMask, EUI_MODAL_CONFIRM_BUTTON } from '@elastic/eui';
+
+import { StartAction } from './use_start_action';
+
+type StartButtonModalProps = Pick<
+  StartAction,
+  'closeModal' | 'isModalVisible' | 'item' | 'startAndCloseModal'
+>;
+export const StartButtonModal: FC<StartButtonModalProps> = ({
+  closeModal,
+  isModalVisible,
+  item,
+  startAndCloseModal,
+}) => {
+  return (
+    <>
+      {isModalVisible && item !== undefined && (
```

Comment:
same as above, this check should happen on a parent level

Reply:
Updated in 4382fee